logbook/_speedups.pyx

```diff
@@ -19,6 +19,15 @@ from cpython.list cimport PyList_New, PyList_Append, PyList_Sort, \
 from cpython.pythread cimport PyThread_type_lock, PyThread_allocate_lock, \
      PyThread_release_lock, PyThread_acquire_lock, WAIT_LOCK
 
+cdef public unsigned short int CRITICAL = 15
```

Comment:
I think it would be wise to add a test comparing the two sets of levels. Otherwise future additions might result in inconsistent levels...